File: web/cognito-config.js
// Reemplaza estos valores con tu AWS Cognito Hosted UI + S3.
window.COGNITO_CONFIG = {
  // Ejemplo: https://tu-dominio.auth.us-east-1.amazoncognito.com
  domain: "domain",
  clientId: "clientId",

  // Region de Cognito y S3
  region: "region",

  // User Pool usado para el Hosted UI (sin region en el valor)
  userPoolId: "userPoolId",

  // Identity Pool para entregar credenciales AWS temporales
  identityPoolId: "us-east-1:XXXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX",

  // Bucket donde guardar fotos
  s3Bucket: "s3Bucket",
  s3Prefix: "folder/",

  // Deben estar permitidas en App client -> Allowed callback/logout URLs
  redirectSignIn: "http://localhost:8080",
  redirectSignOut: "http://localhost:8080",

  // Flujo implicito para ejemplo rapido en web
  responseType: "token",
  scopes: ["openid", "email", "phone"],
};

File: mobile/lib/core/cognito_config.dart
class CognitoConfig {
  const CognitoConfig._();

  static const region = 'us-east-2';
  static const userPoolId = 'us-east-2_1cNEVfoNm';

  // TODO: Replace with your Cognito app client ID.
  static const appClientId = 'appClientId';

  // TODO: Replace with your Cognito Hosted UI domain.
  // Example: myapp.auth.us-east-2.amazoncognito.com
  static const hostedUiDomain = 'hostedUiDomain';

  // Must match callback URLs configured in Cognito app client settings.
  static const redirectScheme = 'dataqubeauth';
  static const redirectHost = 'callback';
  static const redirectUri = '$redirectScheme://$redirectHost';
}
